app/javascript/components/analytics/widgets/GeographicSummary.js

```diff
@@ -94,24 +95,24 @@ class GeographicSummary extends React.Component {
     };
 
     _.takeRight(
-      this.props.stats.monitoree_maps.sort((a, b) => b.day - a.day),
-      MAX_DAYS_OF_HISTORY
+      monitoree_maps.sort((a, b) => b.day - a.day),
+      maxDaysOfHistory
     ).forEach(dayMapsPair => {
       analyticsObject.exposure.push({ date: dayMapsPair.day, value: obtainAnalyticsValue(dayMapsPair.maps, 'Exposure') });
       analyticsObject.isolation.push({ date: dayMapsPair.day, value: obtainAnalyticsValue(dayMapsPair.maps, 'Isolation') });
     });
     return analyticsObject;
   };
 
-  obtainJurisdictionsPermittedToView = () => {
+  obtainJurisdictionsPermittedToView = (monitoree_maps, maxDaysOfHistory) => {
```

Comment:
same thing here with the snake_case. Not a huge deal though

Reply:
I think this one might make more sense to keep as snake_case just because it gets passed on from the back end that way and it's also the name of the model that way we wouldn't have both `monitoree_maps` and `monitoreeMaps` in the same file